llvm/lib/Target/RISCV/RISCVSystemOperands.td

```diff
@@ -486,6 +486,24 @@ def : SysReg<"mctrctl", 0x34e>;
 // Vendor CSRs
 //===-----------------------------------------------
 
+// XSfmclic
+let FeaturesRequired = [{ {RISCV::FeatureVendorXSfmclic} }] in {
+def : SysReg<"mtvt", 0x307>;
+def : SysReg<"mnxti", 0x345>;
+def : SysReg<"mintstatus", 0x346>;
+def : SysReg<"mscratchcsw", 0x348>;
+def : SysReg<"mscratchcswl", 0x349>;
+}
+
+// XSfsclic
+let FeaturesRequired = [{ {RISCV::FeatureVendorXSfsclic} }] in {
+def : SysReg<"stvt", 0x107>;
+def : SysReg<"snxti", 0x145>;
+def : SysReg<"sintstatus", 0x146>;
+def : SysReg<"sscratchcsw", 0x148>;
+def : SysReg<"sscratchcswl", 0x149>;
+}
```

Comment:
I am little hesitate here, we should have sf. prefix here in theory....

Reply:
I'm happy to add this, I didn't because I presumed there exists legacy code which uses these names. If you want, I could add the prefixes to these, and then add a copy as deprecated aliases?

Reply:
We discussed this on the RISC-V LLVM Sync-up today. I will prefix these, and not add deprecated aliases.